File: WebApplication1/Data/Models/Studio.cs
using WebApplication1.Data.DTO;

namespace WebApplication1.Data.Models
{
    public class Studio
    {
        public long ID { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }
        public string Photo { get; set; }

        public virtual List<Movie> Movies { get; set; }

        public Studio() { }

        public Studio(StudioDTO studio)
        {
            Name = studio.Name;
            Description = studio.Description;
            Photo = studio.Photo;
            Movies = new List<Movie>();
           
        }
    }
}

File: TrueMovieAwards/src/app/Models/Studio.ts
import { PartialMovie } from "./PartialMovie";

export class Studio {

    id: number
    name : string
    photo : string
    description : string
    movies : PartialMovie[]

    constructor() {
        this.id = 0;
        this.name = ""
        this.photo = ""
        this.description = ""
        this.movies = []
    }
}